packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -351,6 +373,7 @@ contract VotingV2 is
         if (slash + int256(voterStake.cumulativeStaked) > 0)
             voterStake.cumulativeStaked = uint256(int256(voterStake.cumulativeStaked) + slash);
         else voterStake.cumulativeStaked = 0;
+        emit VoterSlashed(voterAddress, slash);
```

Comment:
I haven't added event for `_updateAccountSlashingTrackers`, I'm hesitant about it as we already have the `VoterSlashed`. They could be reasons for adding a new event for `_updateAccountSlashingTrackers` for off-chain calculations purposes.

Reply:
yup. this is the only place where voters' balance actually go down. we should probs include `voterStake.cumulativeStaked` as well.